dynawo/sources/Modeler/Common/test/TestDelay.cpp

```diff
@@ -297,8 +298,8 @@ compare(const std::vector<std::string>& lhs, const std::vector<std::string>& rhs
 }
 
 TEST(CommonTest, testDelayManagerClassParameters) {
-  std::string formatted1 = "10:1,1.1;2,2.2;3,3.3;4,4.4;5,5.5;";
-  std::string formatted2 = "20:1,1.1;2,2.2;3,3.3;4,4.4;5,5.5;";
+  std::string formatted1 = "10:1.5:1,1.1;2,2.2;3,3.3;4,4.4;5,5.5;";
+  std::string formatted2 = "20:3:1,1.1;2,2.2;3,3.3;4,4.4;5,5.5;";
```

Comment:
evalG is tested in the function but evalMode is not called as `manager.triggerDelay(id2);` is the "same" call it would do. I will modify the test to call evalMode